vignettes/TADAAssessmentUnitUseCase.Rmd

```diff
@@ -0,0 +1,710 @@
+the results that did not meet the standard and review during which years
+and seasons they occurred.
+
+```{r temp analysis and visualization}
+# comparison to standard for temperature
+Temp_Standard <- Filtered_data %>%
+  dplyr::filter(TADA.ComparableDataIdentifier == "TEMPERATURE_NA_NA_DEG C") %>%
+  dplyr::mutate(MonthForAnalysis = lubridate::month(ActivityStartDate),
+                TempStandard = ifelse(MonthForAnalysis %in% c(1,2,3,12), 17.7, 33.7),
+                MeetsStandard = ifelse(TADA.ResultMeasureValue < TempStandard,
+                                       "Yes", "No"))
+
+# create scatterplot for temperature
+Temp_Scatter <- TADA_Scatterplot(Temp_Standard, id_cols = "TADA.ComparableDataIdentifier") 
+
+Temp_Scatter
+
+# create pie chart for temperature
+
+Temp_Pie <- TADA_FieldValuesPie(Temp_Standard, field = "MeetsStandard")
+
+Temp_Pie
+
+# create subset table for temperature
+Temp_Table <- Temp_Standard %>%
+  dplyr::select(MonitoringLocationIdentifier, OrganizationFormalName, ActivityStartDate, TADA.ResultMeasureValue, TempStandard,
+                MeetsStandard)
+
+DT::datatable(Temp_Table)
+```
+
+## Additional Data Exploration
+
+We may also want to use TADA functions for exploration and visualization
+of characteristics without water quality standards. For this example, we
+will use "TOTAL NITROGEN, MIXED FORMS_FILTERED, LAB_NA_MG/L".
+
+To better understand the distribution of results, we can use the TADA
+functions **TADA_Histogram** and **TADA_Boxplot**.
+
+**TADA_Histogram** can be useful for identifying the overall shape of
+the data.
+
+```{r boxplot and histogram}
+
+# filter dataframe to comparable data identifier of interest
+
+Nitrogen_data <- dplyr::filter(Filtered_data, TADA.ComparableDataIdentifier == "TOTAL NITROGEN, MIXED FORMS_FILTERED, LAB_NA_MG/L")
+
+# generate a histogram
+Nitrogen_Histogram <- TADA_Histogram(Nitrogen_data, id_cols = "TADA.ComparableDataIdentifier")
+
+# view histogram
+Nitrogen_Histogram
+```
+
+**TADA_Boxplot** can be useful for identifying skewness and percentiles.
+
+```{r boxplot, fig.width=8, fig.height=6, fig.fullwidth=TRUE}
+Nitrogen_Boxplot <- TADA::TADA_Boxplot(Nitrogen_data, id_cols = "TADA.ComparableDataIdentifier") 
+  
+
+Nitrogen_Boxplot
+```
+
+Filtering the results of **TADA_Stats** for only "TOTAL NITROGEN, MIXED
+FORMS_TOTAL RECOVERABLE_NA_MG/L" and selecting a smaller subset of
+columns or creating a single characteristic scatterplot may also provide
+useful information.
+
+```{r stats and scatterplot}
+# create table with nitrogen stats
+Nitrogen_stats <- Filtered_data_stats %>%
+  dplyr::filter(TADA.ComparableDataIdentifier == "TOTAL NITROGEN, MIXED FORMS_FILTERED, LAB_NA_MG/L") %>%
+  dplyr::select(Location_Count, Measurement_Count, Min, Max, Mean)
+
+DT::datatable(Nitrogen_stats)
+
+# create nitrogen scatterplot
+Nitrogen_Scatterplot <- TADA::TADA_Scatterplot(Nitrogen_data, id_cols = "TADA.ComparableDataIdentifier")
```

Comment:
suggest to be consistent with using (or not using) TADA:: prefix throughout vignette